weatherman/utils.py

```diff
@@ -0,0 +1,44 @@
+import re as regex
+from typing import Tuple
+import os
+
+
+def extract_year_month(date: str) -> Tuple[int, int]:
```

Comment:
I don't see the usage

Reply:
it has usage in other modules of application as specified in PR description